packages/legend-application-query/src/stores/QueryEditorStore.ts

```diff
@@ -506,12 +506,10 @@ export abstract class QueryEditorStore {
     const isValidSearchString =
       searchText.length >= DEFAULT_TYPEAHEAD_SEARCH_MINIMUM_SEARCH_LENGTH;
     try {
-      const searchSpecification = new QuerySearchSpecification();
+      const searchSpecification = QuerySearchSpecification.createDefault(
+        isValidSearchString ? searchText : undefined,
```

Comment:
before searchSpecification.exactMatchName = true;

now with QuerySearchSpecification.createDefault(searchText);
term.exactMatchName = false;
is this expected?

Reply:
same comment as above